Docs/Definitions.md

```diff
@@ -14,6 +14,10 @@ AppArmor is a Linux kernel security module that supplements the standard Linux u
 ## CNI
 CNI (Container Network Interface), a Cloud Native Computing Foundation project, consists of a specification and libraries for writing plugins to configure network interfaces in Linux containers, along with a number of supported plugins. CNI concerns itself only with network connectivity of containers and removing allocated resources when the container is deleted.
 
+## DAG (Directed Acyclic Graph)
+DAG (Directed Acyclic Graph) is a programming style for distributed systems - You can think of it as an alternative to Map Reduce. While MR has just two steps (map and reduce), DAG can have multiple levels that can form a tree structure. Say if you want to execute a SQL query, DAG is more flexible with more functions like map, filter, union etc.
```

Comment:
We don't currently define Map Reduce, probably should if we're going to refer to it